cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt

```diff
@@ -70,6 +74,7 @@ internal interface DataStorageCcpa {
         const val CCPA_DATE_CREATED = "sp.ccpa.key.date.created"
         const val CCPA_SAMPLING_VALUE = "sp.ccpa.key.sampling"
         const val CCPA_SAMPLING_RESULT = "sp.ccpa.key.sampling.result"
+        const val CCPA_EXPIRATION_DATE_MESSAGE = "sp.ccpa.key.expiration.date"
```

Comment:
can we name figure out some sort of a convention for the naming of the constants, for example if we have `sp.ccpa.key.expiration.date`, then the naming will be something like `KEY_CCPA_EXPIRATION_DATE`?

Reply:
@carmelo-iriti was Bohdan's comment addressed? why do we need to store this key in the first place?